### ISSUE TITLE
Training from a checkpoint on new data

### ISSUE THREAD

**Issue by shredpub** [OpenNMT/OpenNMT-py#31]
Hello, I want to fine tune a pretrained model on new data (incremental adaptation) using some new parameters (epochs, learning rate). However, I have been facing problems while using the train_from option due to varying vocabulary sizes (due to different data, hence different preprocessed .pt files, hence different vocab) in both iterations. Could anyone please suggest a smooth way to achieve this? (i.e. make it train on new data from a checkpoint, using existing vocab but a few new parameters)

**Comment by guillaumekln** [comment]
Duplicate of https://github.com/OpenNMT/OpenNMT/issues/195.

Please open an issue on the project you use. Hopefully the solution is the same for both projects here.

**Comment by shredpub** [comment]
Thank you, I am running both projects in parallel and was facing the issue in both. Thanks for your help!

**Comment by shredpub** [comment]
@guillaumekln Actually this works for lua but unfortunately not for the python version.

**Comment by guillaumekln** [comment]
What is the issue?

**Comment by mataney** [comment]
Hey guys, any suggestion what to do for a case like this?
Cheers.

**Comment by eeric** [comment]
make OpenNMT project audio list and sentences text list, as following:
https://github.com/eeric/OpenNMT-py-make-tgt-train.txt/blob/master/openNMT_make_label.py

**Comment by SkyAndCloud** [comment]
> Hey guys, I have a similar problem.
> 
> I want to finetune a pretrained model, but I don't have access to its `.pt` dataset files.
> After running preprocess, can I promise I will not have problems?
> Like a word having different embedding index etc.
> 
> I believe you already thought about this, although `src_vocab` and `tgt_vocab` have no use in the code.
> 
> What do you think?
> Cheers.
Hi, I also encountered this problem. How to solve it now?

**Comment by francoishernandez** [comment]
Should be even easier with v2.0, you just need to provide the same vocab(s) when training.
